README.md

```diff
@@ -2,94 +2,99 @@
 
 <img src="https://github.com/googlesamples/android-architecture/wiki/images/aab-logo.png" alt="Android Architecture Blueprints"/>
 
-The Android framework offers a lot of flexibility when it comes to defining how
-to organize and <em>architect</em> an Android app. This freedom, whilst very valuable, can also result in apps
-with large classes, inconsistent naming and architectures (or lack of) that can
-make testing, maintaining and extending difficult.
+The Android framework provides a lot of flexibility in deciding how to organize and <em>architect</em> an Android app. While this freedom is very valuable, it can also lead to apps with large classes, inconsistent naming schemes, as well as mismatching or missing architectures. These types of issues can make testing, maintaining and extending your apps difficult.
 
-Android Architecture Blueprints is meant to demonstrate possible ways to help
-with these common problems. In this project we offer the same application
-implemented using different architectural concepts and tools.
+The Android Architecture Blueprints project demonstrates strategies to help solve or avoid these common problems. This project implements the same app using different architectural concepts and tools.
 
-You can use these samples as a reference or as a starting point for creating
-your own apps. The focus here is on code structure, architecture, testing and
-maintainability. However, bear in mind that there are many ways to build apps
-with these architectures and tools, depending on your priorities, so these
-shouldn't be considered canonical examples. The UI is deliberately kept simple.
+You can use the samples in this project as a learning reference, or as a starting point for creating your own apps. The focus of this project is on demonstrating how to structure your code, design your architecture, and the eventual impact of adopting these patterns on testing and maintaining your app. You can use the techniques demonstrated here in many different ways to build apps. Your own particular priorities will impact how you implement the concepts in these projects, so you should not consider these samples to be canonical examples. To ensure the focus in kept on the aims described above, the app uses a simple UI.
```

Comment:
The phrase 'is kept' is [passive](https://en.wikipedia.org/wiki/Active_voice), so I suggest keeping it more focused as it is.